src/js/view/view.js

```diff
@@ -826,19 +829,18 @@ function View(_api, _model) {
     };
 
     this.setIntersection = function (entry) {
-        _model.set('intersectionRatio', entry.intersectionRatio);
-        if (_floatOnScroll) {
-            _setFloatingIntersection(entry);
-        }
+        // Round as the IntersectionObserver polyfill sometimes returns ±0.00XXX.
+        const intersectionRatio = Math.round(entry.intersectionRatio * 100) / 100;
+        _model.set('intersectionRatio', intersectionRatio);
     };
 
     function _getCurrentElement() {
         return _model.get('floating') ? _wrapperElement : _playerElement;
     }
 
-    function _setFloatingIntersection(entry) {
-        // Entirely invisible and no floating player already in the DOM
-        const isVisible = entry.intersectionRatio === 1;
+    function _updateFloating() {
```

Comment:
If this is a model "change:" handler, please use the listener args to avoid the extra call to `_model.get`:
```js
function _updateFloating(model, intersectionRatio) {
```